### PR TITLE
Matched pak file system (pfs) files

### PR DESCRIPTION
**Pull request by Mr-Wiseguy**

### CODE REVIEW

File: src/io/pfsinitpak.c [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+    ERRCK(__osContRamWrite(pfs->queue, pfs->channel, 0, temp1, 0));
+    ERRCK(__osContRamRead(pfs->queue, pfs->channel, 0, temp2));
+
+    if (bcmp(temp1, temp2, BLOCKSIZE) != 0) {
+        return PFS_ERR_DEVICE;
+    }
+
+    return __osContRamWrite(pfs->queue, pfs->channel, 0, save, 0);
+}
```
Comment (ethteck) [inline src/io/pfsinitpak.c:101]: nl

File: src/io/pfsselectbank.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    }
+
+    ret = __osContRamWrite(pfs->queue, pfs->channel, 0x8000 / BLOCKSIZE, temp, 0);
+
+    if (ret == 0) {
+        pfs->activebank = bank;
+    }
+
+    return ret;
+}
```
Comment (ethteck) [inline src/io/pfsselectbank.c:20]: new line

**Review by ethteck — CHANGES_REQUESTED** [review]

File: src/io/pfsinitpak.c (hunk 1 above)
Comment (Mr-Wiseguy) [reply src/io/pfsinitpak.c]: Fixed

File: src/io/pfsselectbank.c (hunk 2 above)
Comment (Mr-Wiseguy) [reply src/io/pfsselectbank.c]: During shutdown, all feedback
and control circuitry is turned off. The regulator's output voltage drops to one diode drop below the input voltage.

Comment (Mr-Wiseguy) [reply src/io/pfsselectbank.c]: Fixed

**Pull request merged**